projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html

```diff
@@ -1,170 +1,219 @@
-<div [ngClass]="compact ? 'cx-compact row' : 'row'">
-  <!-- Item Image -->
-  <div class="col-2 cx-image-container">
-    <a
-      [routerLink]="{ cxRoute: 'product', params: item.product } | cxUrl"
-      (click)="viewItem()"
-      tabindex="-1"
-    >
-      <cx-media [container]="item.product.images?.PRIMARY"></cx-media>
-    </a>
-  </div>
-  <!-- Item Information -->
-  <div class="cx-info col-10">
-    <div class="cx-info-container row">
-      <!-- Item Description -->
-      <div [ngClass]="compact ? '' : ' col-md-3 col-lg-3 col-xl-5'">
-        <div *ngIf="item.product.name" class="cx-name">
-          <a
-            class="cx-link"
-            [routerLink]="{ cxRoute: 'product', params: item.product } | cxUrl"
-            (click)="viewItem()"
-            >{{ item.product.name }}</a
-          >
-        </div>
-        <div *ngIf="item.product.code" class="cx-code">
-          {{ 'cartItems.id' | cxTranslate }} {{ item.product.code }}
-        </div>
-        <!-- Generic Configuration-->
+<ng-container *ngIf="item.statusSummaryList[0]?.numberOfIssues > 0">
+  <div [ngClass]="compact ? 'cx-compact row' : 'row'">
+    <div class="cx-error-container">
+      <div [ngClass]="compact ? 'col-2' : 'cx-error-icon-container'">
+        <cx-icon class="cx-error-icon" [type]="iconTypes.ERROR"></cx-icon>
+      </div>
+      <div [ngClass]="compact ? 'col-10' : 'cx-error-msg-container'">
+        {{
+          getIssueMessageKey(item.statusSummaryList[0]?.numberOfIssues)
```

Comment:
No, we need to take the status of the summary list into account! For the textfield configurator, you will have 3 success statuses (which are displayed as errors currently). Probably we should get the issues also from the component

Reply:
Will do

Reply:
done